### DISCUSSION TITLE
can not get motor torque

### DISCUSSION THREAD

**Discussion by lkx8421** [cyberbotics/webots#6041] [Q&A]
**Describe the Bug**
When I use the command "wb_motor_get_torque_feedback()" with "wb_motor_enable_torque_feedback()", it cannot return anything but zero , i don't know why

**Comment by stefaniapedrazzi** [comment]
You can find an example about how to use the `wb_motor_enable_torque_feedback` function in this sample simulation:
https://www.cyberbotics.com/doc/guide/samples-devices#motor-wbt

To open it, select the `samples > devices > motor.wbt` world from the `File > Open Sample World...` Webots menu.

**Reply by lkx8421** [reply]
Thanks for your reply, i copy this sample and in the main functions are `wb_motor_get_torque_feedback()` and `wb_motor_enable_torque_feedback()` ,the `wb_motor_enable_torque_feedback()` applied in the main program and the `wb_motor_get_torque_feedback()` applied in the `while()` circulation, the torque feedback only in the maxstop or minstop position return to the non-zero torque but in the other position is zero torque. And the Webots version in R2023a

**Reply by ShuffleWire** [reply]
could you please post a minimal example (world + controller) that show your issue ?
